### PR TITLE
Format UserGuide

### PR DESCRIPTION
**Pull request by realqijun**
Might conflict with other user guide PRs
Do not approve until resolved.
Solves #132

### CODE REVIEW

**Comment by realqijun** [comment]
UserGuide is ready for review
```
⢀⡴⠑⡄⠀⠀⠀⠀⠀⠀⠀⣀⣀⣤⣤⣤⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀ 
⠸⡇⠀⠿⡀⠀⠀⠀⣀⡴⢿⣿⣿⣿⣿⣿⣿⣿⣷⣦⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠑⢄⣠⠾⠁⣀⣄⡈⠙⣿⣿⣿⣿⣿⣿⣿⣿⣆⠀⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⢀⡀⠁⠀⠀⠈⠙⠛⠂⠈⣿⣿⣿⣿⣿⠿⡿⢿⣆⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⢀⡾⣁⣀⠀⠴⠂⠙⣗⡀⠀⢻⣿⣿⠭⢤⣴⣦⣤⣹⠀⠀⠀⢀⢴⣶⣆ 
⠀⠀⢀⣾⣿⣿⣿⣷⣮⣽⣾⣿⣥⣴⣿⣿⡿⢂⠔⢚⡿⢿⣿⣦⣴⣾⠁⠸⣼⡿ 
⠀⢀⡞⠁⠙⠻⠿⠟⠉⠀⠛⢹⣿⣿⣿⣿⣿⣌⢤⣼⣿⣾⣿⡟⠉⠀⠀⠀⠀⠀ 
⠀⣾⣷⣶⠇⠀⠀⣤⣄⣀⡀⠈⠻⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀ 
⠀⠉⠈⠉⠀⠀⢦⡈⢻⣿⣿⣿⣶⣶⣶⣶⣤⣽⡹⣿⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⠀⠉⠲⣽⡻⢿⣿⣿⣿⣿⣿⣿⣷⣜⣿⣿⣿⡇⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⠀⠀⢸⣿⣿⣷⣶⣮⣭⣽⣿⣿⣿⣿⣿⣿⣿⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⣀⣀⣈⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠇⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⢿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠃⠀⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⠀⠹⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀ 
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠉⠛⠻⠿⠿⠿⠿⠛⠉            
```

File: docs/UserGuide.md [hunk 1]
```diff
@@ -94,13 +94,14 @@ Before continuing, here are some important information you need to know about th
 --------------------------------------------------------------------------------------------------------------------
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer by opening up your terminal (MacOS: Terminal, Windows: Command Prompt), followed by executing `java -version`.
+1. Ensure you have Java `17` or above installed in your Computer by opening up your terminal application (for macOS it's called `Terminal`, and for Windows: `Command Prompt`), followed by executing `java -version`.
 
-1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-T09-1/tp/releases).
+1. Download the latest `.jar` file from [here](https://github.com/AY2425S1-CS2103T-T09-1/tp/releases) (scroll down to assets to find it!).
```
Comment (blanklogic) [inline docs/UserGuide.md:99]: Love the attention to detail towards the User Guide for users that may be new to this method of launching the application.

**Review by blanklogic — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -173,14 +178,18 @@ Format: `add n/NAME e/EMAIL [p/PHONE_NUMBER] [a/ADDRESS] [t/TAG]…​`
 
 <box type="tip" seamless>
 
-**Tip:** A company can have any number of tags (including 0)
+**Tip 1:** A company can have any number of tags (including 0) <br>
+**Tip 2:** A company requires a name and email at the minimum <br>
+**Caution:** Due to the nature of prefix commands, company names are restricted to alphanumeric characters only
```
Comment (uniqly) [inline docs/UserGuide.md:183]: Put this in its caution box, `<box type="warning" seamless>`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -189,24 +198,36 @@ Format: `apply INDEX n/NAME d/DESCRIPTION [as/APPLICATION_STATUS]`
...
+Shows a list of all companies in the address book.
+
+Format: `list`
+
+--------------------------------------------------------------------------------------------------------------------
+
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+<span style="color:red">**Deletes all entries**</span> from the address book.
```
Comment (uniqly) [inline docs/UserGuide.md:225]: Nice that you made this read. 👍 
Another enhancement could be to add a warning box that clearing permanently removes data.

File: docs/UserGuide.md (hunk 2 above)
Comment (uniqly) [reply docs/UserGuide.md:183]: Referring to line 183.

**Review by uniqly — APPROVED** [review]
LGTM!

**Pull request merged**